core/src/main/java/org/sonarsource/sonarlint/core/container/connected/update/ModuleConfigUpdateExecutor.java

```diff
@@ -86,14 +86,8 @@ public void update(String moduleKey) {
 
   private void updateModuleConfiguration(String moduleKey, GlobalProperties globalProps, ServerInfos serverInfos, Path temp) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "serverInfos". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1172)